src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -795,18 +786,7 @@ public Version ProtocolVersion
             }
         }
 
-
-        public int ReadWriteTimeout
-        {
-            get
-            {
-                throw NotImplemented.ByDesignWithMessage(SR.net_PropertyNotImplementedException);
-            }
-            set
-            {
-                throw NotImplemented.ByDesignWithMessage(SR.net_PropertyNotImplementedException);
-            }
-        }
+        public int ReadWriteTimeout { get; set; } = DefaultReadWriteTimeout;
```

Comment:
Should the setter still do parameter validation? E.g. in desktop it throws if `value <= 0 && value != System.Threading.Timeout.Infinite`.

https://github.com/Microsoft/referencesource/blob/90b323fe52bec428fe4bd5f007e9ead6b265d553/System/net/System/Net/HttpWebRequest.cs#L1038-L1040